src/main/java/games/strategy/triplea/TripleAPlayer.java

```diff
@@ -255,7 +255,7 @@ private void tech() {
           + getPlayerBridge().getRemoteDelegate().getClass();
       // for some reason the client is not seeing or getting these errors, so print to err too
       System.err.println(errorContext);
-      ClientLogger.logQuietly(e);
+      ClientLogger.logQuietly(errorContext, e);
       throw new IllegalStateException(errorContext, e);
```

Comment:
Here and at other occurrences we have the same problem you mentioned in another PR. The error is logged and being rethrown, which should really not be the case.

Reply:
Right, but the comment gives some context here.  Apparently, whoever is ultimately catching the exception is not displaying (or logging) it for the user's benefit.  So someone decided to add the logs after-the-fact to ensure the error was displayed in the console.  I purposely didn't change these instances in the prior PR for that reason.

But I agree that we should fix the smell, namely why isn't the final handler reporting the error?  Actually, for all we know, it may do so now, and this comment is out-of-date?